float32 grip_distance
float32 force_threshold
---
bool success
---
float32 current_dist
